frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx

```diff
@@ -156,6 +156,7 @@ export function MissionQueueView({ refreshInterval }: RefreshProps) {
                     robotOptions={Array.from(robotOptions.keys())}
                     echoMissionsOptions={Array.from(echoMissions.keys())}
                     onSelectedMissions={onSelectedEchoMissions}
```

Comment:
im a bit confused here, do we still use the line 158? they have very similar names, can we change to make it more clear or merge them together?

Reply:
We still use 158 since 158 is the one that changes the selected echo missions that are passed on line 159. But I'll rename them